packages/shared/src/__tests__/telemetry.test.ts

```diff
@@ -0,0 +1,89 @@
+import { TelemetryCollector } from '../telemetry';
+
+jest.useFakeTimers();
+
+describe('TelemetryCollector', () => {
```

Comment:
We'll also need to test both environment variables

Reply:
Updated!